src/kite.js

```diff
@@ -491,10 +489,9 @@ export const Kite = {
                 onRemove: () => { this.installing = false; },
               })
               .catch(e => {
-                console.error(e);
-                NotificationsManager.showKiteInstallErrorNotification();
+                NotificationsManager.showKiteInstallErrorNotification(e);
```

Comment:
nit: I'd consider having `showKiteInstallNotification` try-catch the error and show the error notification instead of making it the caller's responsibility

Reply:
I'd prefer to keep it as is. Looking at it from a MVC lens, `showKiteInstallNotification` should only be responsible for how the notification is rendered, and hooking its buttons with the appropriate actions. The implementation details of the actions, and triggering of new notifications should be controlled and provided by the caller.